ansible_collections/arista/avd/docs/getting-started/avd-aap.md

```diff
@@ -0,0 +1,326 @@
+
+=== "Click on Sync Job"
+
+    - On the left pane, click on `Jobs`.
+      - You should see an inventory sync running or complete.
+    - Click on the inventory sync job.
+
+    ![Inventory job sync](../_media/getting-started/aap-avd/inv-job-sync.png)
+
+=== "Sync job output"
+
+    Towards the bottom of the output, we see that AAP successfully parsed our inventory. In this case, we have loaded nine groups and seven hosts.
+
+    ![Sync job output](../_media/getting-started/aap-avd/inv-job-output.png)
+
+=== "Hosts"
+
+    There are a few locations to view the inventory, but for simplicity, we can take a look by clicking on `Hosts` on the left pane.
+
+    ![View inventory](../_media/getting-started/aap-avd/all-hosts.png)
+
+## Job templates and workflow templates
+
+One thing that may need some clarification is the naming of "job templates." These map to playbooks within our project. There is also an option to build workflow templates: a series of job templates with some control logic built-in. For this example, we will use a job template to build and deploy our node configurations with CVP.
+
+=== "Click on Templates"
+
+    On the left pane, click on `Templates`.
+
+    ![Click on Templates](../_media/getting-started/aap-avd/select-templates.png)
+
+=== "Templates - Add"
+
+    - On the center pane, select `Add`.
+    - In the drop down select `Add job template`.
+
+    ![Add template](../_media/getting-started/aap-avd/add-template.png)
+
+=== "Templates - Job"
+
+    The job template is where we leverage the custom execution environment. Since our setup requires specific Ansible collections and Python packages installed, we would like to use a pre-packaged environment with that software. We can modify a decent number of settings, and they may look familiar from previous history with Ansible configurations. If you need a refresher on these options, please see the [official documentation](https://docs.ansible.com/automation-controller/latest/html/userguide/job_templates.html). Once you are happy with the settings, click `Save`.
+
+    !!! warning
+        The playbook is set to "Run," and the EOS instances in use will be changed. Please ensure you are leveraging nonproduction instances when testing.
+
+    ![Template save](../_media/getting-started/aap-avd/template-save.png)
+
+### Surveys
+
+With most jobs, we need a way to authenticate to our CVP instance or EOS nodes. AAP provides a multitude of ways to define credentials. Some options are credentials for Network devices, Container registries, HashiCorp Vault, etc. Feel free to explore any option you need for your environment. For this guide, we will leverage a survey secret. Surveys allow us to ask users for information we require for a job to execute correctly. We will leverage a survey with a default secret value (this is the credential used to connect to our nodes).
```

Comment:
This is a bit of an unconventional choice for a production environment. Normally users would never go into AAP to run, it would be triggered with a webhook, so credentials should be in the credential store.
For a lab or more adhoc things I can see this working fine, but just saying it's not the most realistic choice.

Reply:
I tried to hint at this by mentioning the additional authentication workflows